tests/manifest.jsonld

```diff
@@ -854,6 +855,28 @@
       "computationalComplexity": "high",
       "approval": "rdft:Proposed",
       "action": "rdfc10/test074-in.nq"
+    },
+    {
+      "id": "#test075c",
+      "type": "rdfc:RDFC10EvalTest",
+      "name": "poison – evil (1)",
+      "comment": "A poison graph which is computable given defined limits. (Uses SHA-384)",
```

Comment:
Is "computable given defined limits" a poison graph?  This PR is mainly about the hash algorithm, is a poison graph needed here?  Perhaps have simple input tests for this purpose.  Two tests with the same input, different only by specified hash algorithm, and that result in different outputs (pick an input that does differ).  Trying not to mix test purposes here.

Reply:
I think the poison graphs can come in degrees, leading to an exponential explosion in computational/memory resources. the tests which pass are the same kind of tests as those that blow up, just of a lower degree.

But, as @yamdan mentions, the use of this specific tests is inappropriate for also testing the blank node map output, as it is ambiguous, so I'll change to another. The idea was to use an existing test that would generate different outputs depending on the hash algorithm.